manifests/p/Pretix/PretixSCAN/1.15.1/Pretix.PretixSCAN.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Pretix.PretixSCAN
+PackageVersion: 1.15.1
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://marketplace.pretix.eu/download/pretixscan-windows/482
+  InstallerSha256: 863FAD54E18C9CF4F79125194B232DAD48D38764AA6F1E34600E9415B36C4EE1

```

```suggestion
  InstallerSha256: 863FAD54E18C9CF4F79125194B232DAD48D38764AA6F1E34600E9415B36C4EE1
  ProductCode: pretixSCAN
```
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\pretixSCAN`